src/main/scala/com/wavesplatform/network/CoordinatorHandler.scala

```diff
@@ -43,9 +43,9 @@ class CoordinatorHandler(checkpointService: CheckpointService,
   private val processBlock = Coordinator.processSingleBlock(checkpointService, history, blockchainUpdater, time, stateReader, utxStorage, settings.blockchainSettings, featureProvider) _
   private val processMicroBlock = Coordinator.processMicroBlock(checkpointService, history, blockchainUpdater, utxStorage) _
 
-  private def scheduleMiningAndBroadcastScore(score: BigInt): Unit = {
+  private def scheduleMiningAndBroadcastScore(reason: LocalScoreChanged.Reason = LocalScoreChanged.Reason.Other)(score: BigInt): Unit = {
```

Comment:
What's the point of using  default value?

Reply:
To not break extension loading